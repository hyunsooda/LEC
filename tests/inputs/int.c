#include <stdio.h>
#include <stdlib.h>

int main() {
    char SIZE = 5;
    int* intarr = (int*)malloc(sizeof(int) * SIZE);
    intarr[4] = 123;
    printf("%d\n", intarr[7]);
    exit(1);
}
