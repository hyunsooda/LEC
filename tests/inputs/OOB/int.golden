B[31mFound out of bound access: ["int.c":10:20]B[37m
    array length: 15, indexed by: 56
    variable name: "intarr", allocated at: 8
